ReactiveCocoaTests/UIKit/UIButtonSpec.swift

```diff
@@ -7,6 +7,8 @@ import enum Result.NoError
 
 class UIButtonSpec: QuickSpec {
 	override func spec() {
+		UIControl.swizzleForTesting()
```

Comment:
This isn't the best solution. But I am not aware of any nice way to "inject" this at startup. 😕